.settings/com.googlecode.goclipse.core.prefs

```diff
@@ -0,0 +1,2 @@
+build_targets=<?xml version\="1.0" encoding\="UTF-8"?><build_targets xmlns\="com.googlecode.goclipse.core">\r\n<target auto_enabled\="true" config\="build" exe_path\="${project_loc\:alm.exe}" n_enabled\="true" version2\="true">\r\n<command_invocation append_env\="true" command_arguments\="${GO_TOOL_PATH} install -v -gcflags &quot;-N -l&quot; github.com/almighty/almighty-core/...">\r\n<env_vars/>\r\n</command_invocation>\r\n</target>\r\n<target auto_enabled\="false" config\="build-tests" n_enabled\="false" version2\="true"/>\r\n<target auto_enabled\="false" config\="lint" n_enabled\="false" version2\="true"/>\r\n<target auto_enabled\="false" config\="[run-tests]" n_enabled\="false" version2\="true"/>\r\n</build_targets>

```

Comment:
Agreed.

Konrad Kleine | Software Engineer
Red Hat GmbH
phone: +4921195439689
email: kkleine@redhat.com

On Thu, Jun 30, 2016 at 9:24 AM, Thomas Mäder <notifications@github.com>
wrote:

> In .settings/com.googlecode.goclipse.core.prefs
> <https://github.com/almighty/almighty-core/pull/4#discussion_r69083280>:
>
> > @@ -0,0 +1,2 @@
> > +build_targets=<?xml version\="1.0" encoding\="UTF-8"?><build_targets xmlns\="com.googlecode.goclipse.core">\r\n<target auto_enabled\="true" config\="build" exe_path\="${project_loc\:alm.exe}" n_enabled\="true" version2\="true">\r\n<command_invocation append_env\="true" command_arguments\="${GO_TOOL_PATH} install -v -gcflags &quot;-N -l&quot; github.com/almighty/almighty-core/...">\r\n<env_vars/>\r\n</command_invocation>\r\n</target>\r\n<target auto_enabled\="false" config\="build-tests" n_enabled\="false" version2\="true"/>\r\n<target auto_enabled\="false" config\="lint" n_enabled\="false" version2\="true"/>\r\n<target auto_enabled\="false" config\="[run-tests]" n_enabled\="false" version2\="true"/>\r\n</build_targets>
>
> .settings should not be in git as it is eclipse specific
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/almighty/almighty-core/pull/4/files/6809e69838cf74a34dc19ee246ed4f57b66d4168#r69083280>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AALzgOMmcWlamT8D2kMasFwfUwfZzcWoks5qQ29BgaJpZM4I8524>
> .
>